Lib/importlib/abc.py

```diff
@@ -199,6 +208,11 @@ class SourceLoader(_bootstrap_external.SourceLoader, ResourceLoader, ExecutionLo
 
     def path_mtime(self, path):
         """Return the (int) modification time for the path (str)."""
+        import warnings
+        warnings.warn('SourceLoader.path_mtime is deprecated in favour of '
+                      'SourceLoader.path_stats().',
+                      DeprecationWarning,
+                      stacklevel=2)
```

Comment:
```suggestion
                      DeprecationWarning, stacklevel=2)
```

Remain consistent with other additions.